pylmod/tests/test_gradebook.py

```diff
@@ -110,6 +110,195 @@ class TestGradebook(BaseTest):
         ]
     }
 
+    OPTIONS_BODY = {
```

Comment:
Same with this data structure, if we don't have code that is validating any of this data, other than it's equivalency, you can use the simple structs and just create it in the test.  Also, if you are only using it in one test, it should be declared in that test, so this should just be something like:
```python
  @httpretty.activate
    def test_get_options(self):
        """Verify that we can get the options for a gradebook."""
        test_body = {'data': { 'membershipQualifier': '/project/mitxdemosite'' }}
        httpretty.register_uri(
            httpretty.GET,
            '{0}gradebook/options/{1}'.format(
                self.GRADEBOOK_REGISTER_BASE,
                self.GRADEBOOK_ID
            ),
            body=json.dumps(self.test_body)
        )
        gradebook = GradeBook(self.CERT, self.URLBASE, self.GBUUID)
        options = gradebook.get_options(gradebook_id=self.GRADEBOOK_ID)
        self.assertEqual(
            options['membershipQualifier'],
            test_body['data']['membershipQualifier']
        )
```
instead of having a class level static structure and URL register.